canvas_account_admin_tools/migrations/0002_school_permissions_20160826_1252.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import itertools
+
+from django.conf import settings
+from django.db import migrations
+
+from lti_permissions.models import DEFAULT_ROLES, DEFAULT_SCHOOLS
+
+
+# intialize permissions for all apps to the defaults (TLT-2650)
+PERMISSION_NAMES = settings.TOOL_PERMISSIONS.values()
+SCHOOL_PERMISSION_DATA = itertools.product(
+    PERMISSION_NAMES,
+    DEFAULT_SCHOOLS,
+    DEFAULT_ROLES)
+
+
+def create_school_permissions(apps, schema_editor):
+    SchoolPermission = apps.get_model('lti_permissions', 'SchoolPermission')
```

Comment:
As suggested by @cmurtaugh, perhaps we can:
1. add account admin permissions (default)
2. figure out the additional permissions based on what's in the `LtiPermission` table and automatically generate those `SchoolPermission`s.